src/RTCIceCandidate.ts

```diff
@@ -1,14 +1,22 @@
+interface RTCIceCandidateInfo {
+    candidate?: string;
+    sdpMLineIndex?: number | null;
+    sdpMid?: string | null;
+}
+
 export default class RTCIceCandidate {
     candidate: string;
     sdpMLineIndex: number;
     sdpMid: string;
 
-    constructor({ candidate = '', sdpMLineIndex = null, sdpMid = null }) {
-        if (sdpMLineIndex === null || sdpMid === null) {
+    constructor(info: RTCIceCandidateInfo) {
+        const { candidate, sdpMLineIndex, sdpMid } = info;
+        // == null checks for both null and undefined, but not for other falsy values
+        if (sdpMLineIndex == null || sdpMid == null) {
             throw new TypeError('`sdpMLineIndex` and `sdpMid` must not null');
         }
 
-        this.candidate = candidate;
+        this.candidate = candidate ?? '';
```

Comment:
Is this necessary? Doesn't the default value above cover it?

Reply:
There's no default value anymore. 

Please note that I added an other typing to this PR, even though it doesn't match the branch name etc...